Yoast/javascript
# LINGO-349 Add Hungarian stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements Hungarian stemmer.
* [Yoast SEO Premium] Introduces word form support for Hungarian.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

In Yoast SEO Premium:
* Change site language to Hungarian
* Add a Hungarian text of at least 300 words. (You can use Wikipedia as a source)
* Set `mindennapjaim` as keyphrase and enter `mindennap` in the text/meta description/title. The relevant assessments should recognise `mindennap` as an instance of the keyphrase. You should see the following assessment feedback:
Keyphrase in meta description: `"Keyphrase or synonym appear in the meta description. Well done!"`
Keyphrase density: `"The focus keyphrase was found 1 time. That's less than the recommended minimum of 3 times for a text of this length. Focus on your keyphrase!"` (Number depending on how often you added the keyphrase in the text)
Keyphrase in title: `"Does not contain the exact match. Try to write the exact match of your keyphrase in the SEO title and put it at the beginning of the title."`
* Switch the keyphrase and the form in the text/meta description/title around and make sure the assessment results are still the same.
* Repeat the same process with `más` as keyphrase and `másoké` in the text/meta description/title.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-349
